builtin/providers/aws/resource_aws_autoscaling_group.go

```diff
@@ -244,7 +244,13 @@ func resourceAwsAutoscalingGroup() *schema.Resource {
 				},
 			},
 
-			"tag": autoscalingTagsSchema(),
+			"tag": autoscalingTagSchema(),
+
+			"tags": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem:     &schema.Schema{Type: schema.TypeMap},
+			},
```

Comment:
Actually we do have a test for it, just in a different file I was expecting and it _is_ failing on this branch:

```
=== RUN   TestAccAWSAutoScalingGroup_importBasic
--- FAIL: TestAccAWSAutoScalingGroup_importBasic (206.37s)
	testing.go:280: Step 1 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.

		(map[string]string) {
		}


		(map[string]string) (len=13) {
		 (string) (len=5) "tag.#": (string) (len=1) "1",
		 (string) (len=18) "tag.4237349706.key": (string) (len=7) "FromTag",
		 (string) (len=34) "tag.4237349706.propagate_at_launch": (string) (len=4) "true",
		 (string) (len=20) "tag.4237349706.value": (string) (len=6) "value1",
		 (string) (len=6) "tags.#": (string) (len=1) "2",
		 (string) (len=8) "tags.0.%": (string) (len=1) "3",
		 (string) (len=10) "tags.0.key": (string) (len=9) "FromTags1",
		 (string) (len=26) "tags.0.propagate_at_launch": (string) (len=1) "1",
		 (string) (len=12) "tags.0.value": (string) (len=6) "value2",
		 (string) (len=8) "tags.1.%": (string) (len=1) "3",
		 (string) (len=10) "tags.1.key": (string) (len=9) "FromTags2",
		 (string) (len=26) "tags.1.propagate_at_launch": (string) (len=1) "1",
		 (string) (len=12) "tags.1.value": (string) (len=6) "value3"
		}
```